### ISSUE TITLE
Nuget package version mismatch

### ISSUE THREAD

**Issue by dahall** [dahall/TaskScheduler#124]
I just downloaded the latest packages from Nuget:Task Scheduler Managed Wrapper UI LibraryTask Scheduler Managed WrapperTimeSpan Helper LibraryWindows Forms Aero-Wizard ControlWinForms Group ControlsWhen I compile, I get:Assembly 'Microsoft.Win32.TaskSchedulerEditor, Version=1.9.0.23358, Culture=neutral, PublicKeyToken=7be0d908c369d1a5' uses 'GroupControls, Version=1.3.5.20798, Culture=neutral, PublicKeyToken=3726d9da21b6ac8f' which has a higher version than referenced assembly 'GroupControls, Version=1.3.5.14280, Culture=neutral, PublicKeyToken=3726d9da21b6ac8f'And sure enough, the current Group Control package in Nuget has version 1.3.5.14280Can you please sync the Nuget packages version?Thanks,/M

Originally posted: 2012-11-26T14:33:11

**Comment by dahall** [comment]
I had already found this and will be correcting it with the 1.9.1 version push in the next few days. Sorry for the error.

Originally posted: 2012-11-27T10:26:08

**Comment by dahall** [comment]
Great, hope you'll also take care of the Aero version inconsistency?

If I (from Nuget) install Task Scheduler Managed Wrapper UI, it will install the Aero components automatically. However, Nuget will find a newer Aero component and prompt me to update, which isn't possible, it will fail on saying it can't match any dependency to the TS Editor component.

If I , on the other and first install the latest Aero component, TSMW will fail saying "Already referencing a newer version of 'AeroWizard'."

In short, a complete version mess.

May I suggest that you skip the auto build version increment ".*" in your AssemblyInfo.cs files in all your related projects and set them manually instead? That would most likely solve these issues.

Thanks.


Originally posted: 2012-11-27T15:10:56

**Comment by dahall** [comment]
Fixed today with download version 1.9.1 and update to NuGet.

Originally posted: 2012-11-29T13:44:13
